utils/docker/Dockerfile-rpmbuild.sles.12.3

```
@@ -80,7 +80,7 @@ RUN zypper --non-interactive ar --gpgcheck-allow-unsigned -f ${JENKINS_URL}job/d
 RUN zypper --non-interactive install dpdk-devel fio-src libiscsi-devel cunit-devel
 
 # meson
-RUN zypper --non-interactive install python-rpm-macros python3-base
+RUN zypper --non-interactive install python-rpm-macros python3-base python-devel python3-devel
```

I could be wrong, but I'd imagine `python3-devel` depends on `python3-base`.  If so:
```suggestion
RUN zypper --non-interactive install python-rpm-macros python-devel python3-devel
```

to avoid specifying too many packages.